src/egon/data/datasets/chp/__init__.py

```diff
@@ -322,6 +322,12 @@ def insert_chp_statusquo():
         WORKING_DIR_MASTR_NEW / "bnetza_mastr_combustion_cleaned.csv"
     )
 
+    mastr_biomass = pd.read_csv(
+        WORKING_DIR_MASTR_NEW / "bnetza_mastr_biomass_cleaned.csv"
+    )
+
+    mastr = pd.concat([mastr, mastr_biomass]).reset_index(drop=True)
+

```

Comment:
Did you also increase the version number of the CHP dataset?

Reply:
Done in src/egon/data/datasets/chp/__init__.py